### PR TITLE
Update README.md

### LINKED ISSUE #1: New README
@dawnofmidnight I believe you have the power to do this, just add a quick bot description

### PR DESCRIPTION
**Pull request by hypergamer80**
Closes #1.

JUST APPROVE THIS

*Edited by dawnofmidnight*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,10 @@
-ᓚᘏᗢ 
+O F F I C I A L   M A S C O T   F O R   T H E   S U P R E M E   ᓚᘏᗢ
+O F F I C I A L   M A S C O T   F O R   T H E   S U P R E M E   ᓚᘏᗢ
+O F F I C I A L   M A S C O T   F O R   T H E   S U P R E M E   ᓚᘏᗢ
+O F F I C I A L   M A S C O T   F O R   T H E   S U P R E M E   ᓚᘏᗢ
+O F F I C I A L   M A S C O T   F O R   T H E   S U P R E M E   ᓚᘏᗢ
+O F F I C I A L   M A S C O T   F O R   T H E   S U P R E M E   ᓚᘏᗢ
```
Comment (dawnofmidnight) [inline README.md:6]: Add something in the middle about how it's out community bot. Then I'll approve

**Review by dawnofmidnight — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Kronifer) [reply README.md:6]: +1

File: README.md [hunk 2]
```diff
@@ -1 +1,10 @@
...
+O F F I C I A L   M A S C O T   F O R   T H E   S U P R E M E   ᓚᘏᗢ
```
Comment (Kronifer) [inline README.md]: Maybe don't repeat this insessently it looks unprofessional

File: README.md (hunk 1 above)
Comment (hypergamer80) [reply README.md]: resolved.

**Review by Kronifer — CHANGES_REQUESTED** [review]
I've a few nitpicks

File: README.md (hunk 2 above)
Comment (dawnofmidnight) [reply README.md]: I think remove the words. Just spam cats

Comment (Kronifer) [reply README.md]: Yeah have a line of cats at the bottom

**Comment by dawnofmidnight** [comment]
No words besides the serious ones. Only cats.

**Review by Kronifer — CHANGES_REQUESTED** [review]
Remove last 3 lines, then lgtm

**Comment by dawnofmidnight** [comment]
Can it just be
```
This is the official community mascot bot for Catlang, the supreme cat-oriented programming langauge with a revolutionnary programming paradigm.

ALL HAIL THE DIVINE ᓚᘏᗢ!

ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ ᓚᘏᗢ 
```?

**Review by dawnofmidnight — APPROVED** [review]
Much better. @Kronifer, How does this look to you?

**Review by Kronifer — APPROVED** [review]
Looks good!

**Pull request merged**